src/python/pants/backend/python/util_rules/pex_cli.py

```diff
@@ -20,19 +20,29 @@
 from pants.engine.process import Process
 from pants.engine.rules import Get, collect_rules, rule
 from pants.util.frozendict import FrozenDict
-from pants.util.meta import frozen_after_init
+from pants.util.meta import classproperty, frozen_after_init
 
 
 class PexBinary(ExternalTool):
     """The PEX (Python EXecutable) tool (https://github.com/pantsbuild/pex)."""
 
     options_scope = "download-pex-bin"
     name = "pex"
-    default_version = "v2.1.14"
-    default_known_versions = [
-        f"v2.1.14|{plat}|12937da9ad5ad2c60564aa35cb4b3992ba3cc5ef7efedd44159332873da6fe46|2637138"
-        for plat in ["darwin", "linux "]
```

Comment:
I just noticed this space ... that can't be right.

Reply:
The space is stripped as a convenience for readability in pants.toml. It doesn't matter, but agreed that's confusing.

Reply:
So I can remove it? I find we don't use the value in our tomls?:
```
$ git grep linux *.toml
$
```

Does it somehow show up in users' tomls?!

Reply:
... and why no space in `"darwin"` - there are a million questions this raises.

Reply:
Yep, you can remove it. It was a typo, but a typo that doesn't break anything.

What I mean with toml files is that users are able to set values like `--protoc-known-versions`. From the help message:

>  Values are space-stripped, so pipes can be indented for readability if necessary.

Reply:
Gotcha - thanks. I may save that for follow up.

Reply:
Actually, this needs to follow #10779 so I'll fix it in the PR as soon as that PR lands.